### ISSUE TITLE
Problem: "Please download the chrome app"

### ISSUE THREAD

**Issue by tdrkmn** [bp2008/DahuaLoginBypass#5]
Hi!
As the title suggests, I get a popup with the message "Please download the chrome app" when I try to log in.
![image](https://github.com/bp2008/DahuaLoginBypass/assets/13338579/b1ffe9c0-8fd9-46c4-93cb-040a96a305ab)

I'm confused, isn't the extension for chrome? Anything else needed?

My Chrome version is 124.0.6367.79 and I installed DahuaLoginBypass v4.

**Comment by bp2008** [comment]
Hi.  That message box is not from this extension.  I believe that message is simply coming from the camera.  Unfortunately it is not clear about what it wants you to install.  I remember this from ages ago but I'm pretty sure it does not work anymore: https://ipcamtalk.com/threads/chrome-app-for-dahuas.13429/

**Comment by straycodelp** [comment]
Yes, I know it isn't a message from the extension. But I was hoping there was a solution. So... Do you know if the problem is related to chrome version?

Anyway I think the implementation can be ported to Firefox extensions that doesn't have this problem.
